OpenRA.Mods.Common/AI/States/GroundStates.cs

```diff
@@ -20,6 +20,14 @@ protected virtual bool ShouldFlee(Squad owner)
 		{
 			return base.ShouldFlee(owner, enemies => !AttackOrFleeFuzzy.Default.CanAttack(owner.Units, enemies));
 		}
+
+		protected Actor FindClosestEnemy(Squad owner)
+		{
+			if (!owner.IsValid)
+				return null;
```

Comment:
No need to repeat this check in the helper, the callers all need to check it anyway.